client/src/components/RecommendedAlbums.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+
+class RecommendedAlbums extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeAlbum: null
+    }
+    this.enterHandler = this.enterHandler.bind(this);
+    this.leaveHandler = this.leaveHandler.bind(this);
+  }
+
+  enterHandler(albumID) {
+    this.setState({activeAlbum: albumID})
+  }
+
+  leaveHandler() {
+    this.setState({ activeAlbum: null })
+  }
+
+  render() {
+    return (
+      this.props.albums.map((album => {
+        return <div key={album.id} className="album-square" onMouseEnter={() => this.enterHandler(album.id)} onMouseLeave={this.leaveHandler}> <img className="album-art" src={album.albumArt} />
```

Comment:
You can have props on new lines without affecting the render while enhancing readability.